examples/Cargo.toml

```
@@ -52,7 +52,7 @@ inferno = "0.11.6"
 tempfile = "3.3.0"
 
 # opentelemetry example
-opentelemetry = { version = "0.18.0", default-features = false, features = ["trace"] }
+opentelemetry = { version = "0.19.0", default-features = false, features = ["trace"] }
 opentelemetry-jaeger = "0.17.0"
```

`opentelemetry-jaeger` needs updating as well:
```suggestion
opentelemetry-jaeger = "0.18.0"
```

See [changelog](https://github.com/open-telemetry/opentelemetry-rust/blob/879d6ff9d251db158da7a5720ff903fa6cb7c1e2/opentelemetry-jaeger/CHANGELOG.md)